.github/workflows/ci-nx-dev-e2e.yml

```diff
@@ -0,0 +1,71 @@
+name: CI nx-dev e2e
```

Comment:
Can we remove changes to `nx-dev-e2e` please?

We need to re-enable the tests but we shouldn't do it in this PR.

Reply:
I've split out nx-dev-e2e to https://github.com/nrwl/nx/pull/10088